pom.xml

```diff
@@ -5,149 +5,79 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.11</version>
-    <relativePath />
+    <version>3.26</version>
```

Comment:
What does this bump mean? Will we be limited in the versions of jenkins that the plugin will be suitable for?

Reply:
It's a parent pom with defined project dependencies, versions etc. Jenkins version will be defined via pom property `jenkins.version` (mandatory parameter) => so we'll be not limited to the versions of jenkins that our plugin will be suitable for.

It's recommended to update the version of parent pom (because of fixes or supporting new features etc)